packages/vm/tests/retesteth/clients/ethereumjs/genesis/London.json

```diff
@@ -0,0 +1,10 @@
+{
+    "params" : {
+        "fork" : "London",
```

Comment:
Note: at this point, these configs do not do anything, we read them from our own `config.ts` file.